web/components/application/hacker/pages/PageOne/PageOne.js

```diff
@@ -56,6 +83,108 @@ class PageOne extends Page {
+          name="gender"
+          label="What gender do you identify as?"
+          className="margin-ends-giga"
+          onChange={({ value: newGender }) => this.updateApplication({ gender: newGender })}
+          onBlur={() => this.setFieldAsBlurred('gender')}
+          error={this.getErrorIfBlurred('gender')}
+          isSearchable
+          allowNewOption
+          />
+        <RadioGroup
+          name="age-19-or-older"
+          label="Will you be 19 years of age or older by January 26th, 2019?"
+          className="dir-row margin-ends-giga"
+          onChange={(newIsAdult) => {
+            newIsAdult = newIsAdult === 'true';
+            this.updateApplication({ isAdult: newIsAdult });
+          }
+          }
+          >
+          <RadioButton
+            label="Yes"
+            value="true"
+            selected={isAdult === true}
+            />
+          <RadioButton
+            label="No"
+            value="false"
+            selected={isAdult === false}
+            />
+        </RadioGroup>
+        <Select
+          options={levelsOfEducation}
+          value={education}
+          name="education-level"
+          label="What level of education are you currently studying at?"
+          className="margin-ends-giga"
+          onChange={({ value: newEducation }) => this.updateApplication({
+            education: newEducation,
+          })}
+          onBlur={() => this.setFieldAsBlurred('education')}
+          error={this.getErrorIfBlurred('education')}
+          />
+        <TextInput
+          label="What is your graduation year?"
+          name="graduation-year"
+          className="margin-ends-giga"
+          value={gradYear}
+          onChange={newGradYear => this.updateApplication({ gradYear: newGradYear })}
+          onBlur={() => this.setFieldAsBlurred('gradYear')}
+          error={this.getErrorIfBlurred('gradYear')}
+          />
+        <Select
+          options={schools}
+          value={school}
+          name="school-name"
+          label="What school do you currently attend?"
+          className="margin-ends-giga"
+          onChange={({ value: newSchool }) => this.updateApplication({ school: newSchool })}
+          onBlur={() => this.setFieldAsBlurred('school')}
+          error={this.getErrorIfBlurred('school')}
+          isSearchable
+          allowNewOption
+          />
+        <SelectCity
+          inputValue={cityQuery}
+          value={city}
+          onChange={newCityQuery => this.setState({ cityQuery: newCityQuery })}
+          onSelect={newCity => this.updateApplication({ city: newCity })}
+          onBlur={() => this.setFieldAsBlurred('city')}
+          error={this.getErrorIfBlurred('city')}
+          className="margin-ends-giga"
+          />
+        <p>
+          {'Would you like to apply for a travel reimbursement? If you are coming from the University of Washington, or the surrounding area, we will be providing shuttle busses.'}
```

Comment:
LOOOL nice, looks like making this string a JS expression this avoids the annoying eslint max length rule. Wish I knew about this earlier when doing the FAQ questions